rotkehlchen/chain/bitcoin/utils.py

```diff
@@ -264,3 +273,12 @@ def scriptpubkey_to_btc_address(data: bytes) -> BTCAddress:
         return scriptpubkey_to_p2sh_address(data)
 
     return scriptpubkey_to_bech32_address(data)
+
+
+def derive_p2pkh_from_p2pk(pubkey_hex: str) -> BTCAddress | None:
```

Comment:
This seems to never return None. Raising an exception on Error means this will raise (which should also be mentioned in the docstring) rather than return None in case of error.  This is bad, as mypyu will now make you handle `None` in all places you call it, and that will be useless since this will never return None. ... but will raise which you won't handle.

Please fix this.